csatar-plugins/forms/components/basicForm/partials/form.htm

```diff
@@ -0,0 +1,84 @@
+<form>
+    <div class="fs-nav">
+        <div class="control-toolbar">
+            <div class="toolbar-item toolbar-primary">
+                <div data-control="toolbar">
+                    <div id="fs-saving-loader" style="float:left; display:none; width: 150px;" class="loading-indicator-container">
+                        <div class="loading-indicator size-small" style="background:none;">
+                            <span></span>
+                            <div>Saving ...</div>
+                        </div>
+                    </div>
+
+                    <div id="fs-closing-loader" style="float:left; display:none; width: 150px;" class="loading-indicator-container">
+                        <div class="loading-indicator size-small" style="background:none;">
+                            <span></span>
+                            <div>Closing ...</div>
```

Comment:
Ez így nem fordítható és az már biztos, hogy nekünk ez magyarul is fog kelleni.
A `csatar-plugins/forms/controllers/forms/` könyvtárban nagyon jó példák vannak.
(Ne felejtsük el, hogy az volt az elképzelés, hogy ez egy olyan plugin lesz, amit máshol is fel tudunk használni)